src/components/PlanRoom/Board.tsx

```diff
@@ -0,0 +1,20 @@
+import { Box } from '@chakra-ui/react';
+import { DndProvider } from 'react-dnd';
+import { HTML5Backend } from 'react-dnd-html5-backend';
+import Items from './RoomTables/Item';
+
+const Board = () => (
+  <Box
+    id="board"
+    h="400px"
```

Comment:
In my opinion we should avoid using static `px` definitions. 
Views should be responsive based on browser width and height.
Consider using modifiers like `rem` or/and `em`.

Reply:
I think so too, but for calculations later, the static determination of the size gives the possibility of defining the framework. Already such a problem with responsiveness to width arises. Then, it is difficult to give a frame to which the conditions will apply

![image](https://user-images.githubusercontent.com/43721352/120540291-cd5a2c80-c3e8-11eb-92ae-1a87ae1e2b01.png)

Example that size looks good on smaller devices. I would leave it at that